subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformIntegrationTest.groovy

```diff
@@ -163,6 +163,193 @@ class ArtifactTransformIntegrationTest extends AbstractIntegrationSpec {
         file("app/build/libs").listFiles().size() == 1
     }
 
+    def "User gets a reasonable error message when a transformation throws exception"() {
+        given:
+        executer.withStackTraceChecksDisabled()
```

Comment:
Should remove this. If the exception is propagated out of the transformer instead of being logged, the exception will be reported correctly and the stack trace check does not need to be disabled.